contents/blog/best-hotjar-alternatives.md

```diff
@@ -0,0 +1,431 @@
+
+According to [BuiltWith](https://trends.builtwith.com/analytics/PostHog), as of February 2024, PostHog is used on 5,169 (0.52%) of the top 1 million websites. Hotjar is used by [72,048 of the top 1 million websites](https://trends.builtwith.com/analytics/Microsoft-Clarity). Typical PostHog users are engineers and product managers at startups and mid-size companies, such as [Webshare](/customers/webshare), [AssemblyAI](/customers/assemblyai), and [Purplewave](/customers/purplewave).
+
+#### Key features
+
+- **Session replays:** Including event timelines, console logs, network activity.
+- **Heatmaps:** Visualize user and rage clicks anywhere on your site.
+- **Product analytics:** Analyze funnels, user paths, retention, and trends.
+- **Feature flags & A/B testing:** Rollout features safely, run multivariate A/B tests.
+- **Surveys:** Run customer satisfaction, NPS, and PMF surveys.
+
+### How does PostHog compare to Hotjar?
+
+According to BuiltWith, 27% of sites using Hotjar also use an analytics tool like FullStory (21% of sites use Mixpanel). PostHog can replace them all and has a [generous free tier](/pricing) of 5,000 sessions and 1 million analytics events per month.
+
+<ComparisonTable column1="PostHog" column2="Hotjar">
+  <ComparisonRow column1={true} column2={true} feature="Web session replay" description="View real sessions on websites and web apps" />
+  <ComparisonRow column1="Android only*" column2={false} feature="Mobile session replay" description="View real sessions in mobile apps" />
+  <ComparisonRow column1={true} column2={true} feature="Event timeline" description="History of everything that happened in a user's session" />
+  <ComparisonRow column1={true} column2={false} feature="Network monitor" description="Analyze performance and network calls" />
+  <ComparisonRow column1={true} column2={true} feature="Console logs" description="Debug issues faster by browsing the user's console" />
+  <ComparisonRow column1={true} column2={true} feature="Heatmaps" description="See where users click and interact" />
+  <ComparisonRow column1={true} column2={true} feature="Rage clicks" description="Detect when users click rapidly in the same place" />
+  <ComparisonRow column1={false} column2={true} feature="Scroll maps" description="See how far users scroll on websites and apps" />
+  <ComparisonRow column1={true} column2={false} feature="Product analytics" description="Custom trends, funnels, paths, and retention analysis" />
+  <ComparisonRow column1={true} column2={false} feature="Feature flags" description="Control feature access and phase rollouts" />  
+  <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Test changes and analyze their impact" />
+  <ComparisonRow column1={true} column2={true} feature="Surveys" description="Collect feedback and run satisfaction surveys" />
+</ComparisonTable>
+
+*iOS replays are under development.
```

Comment:
genuinely didnt realize it was still under development. I was planning to write a tutorial for iOS replays this week 😆

Reply:
Yeah, Android is still beta as well afaik.